packages/adapters/ethers/src/client.ts

```diff
@@ -479,6 +480,19 @@ export class EthersAdapter extends AdapterBlueprint {
     return { profileName: undefined, profileImage: undefined }
   }
 
+  private listenPendingTransactions(provider: Provider) {
+    const browserProvider = new BrowserProvider(provider)
+
+    try {
+      browserProvider.on('pending', () => {
+        this.emit('pendingTransactions')
+      })
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error('Error listening to pending transactions', error)
```

Comment:
Maybe we can use [AlertController](https://github.com/reown-com/appkit/blob/main/packages/core/src/controllers/AlertController.ts) for users who use debug mode ?